src/styles/Document.js

```diff
@@ -54,13 +54,16 @@ export default css`
 .renderer {
   padding: 8px 0;
   display: flex;
-  max-height: 500px;
```

Comment:
This easily can be replace by a CSS variable definition, like:
```css
max-height: var(--api-resource-example-document-max-height, 500px);
```

Reply:
Done